charts/v1.0.0/csi-driver-smb/templates/csi-smb-driver.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: storage.k8s.io/v1beta1
```

Comment:
revert change here since v1.0.0 is released, only need to change `charts/latest/csi-driver-smb/templates/csi-smb-driver.yaml`

Reply:
reverted this change. Thanks for the code review!